File: src/_1ProgrammingBasicsWithJavaMay2023/_03ConditionalStatementsAdvanced/_1Lab/_04PersonalTitles.java
package _1ProgrammingBasicsWithJavaMay2023._03ConditionalStatementsAdvanced._1Lab;

import java.util.Scanner;

public class _04PersonalTitles {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double age = Double.parseDouble(scanner.nextLine());
        String gender = scanner.nextLine();
        if (gender.equals("m")) {
            if (age >= 16) {
                System.out.println("Mr.");
            } else if (age < 16){
                System.out.println("Master");
            }
        } else if (gender.equals("f")) {
            if (age >= 16) {
                System.out.println("Ms.");
            } else if (age < 16) {
                System.out.println("Miss");
            }
        }
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_3ConditionalStatementsAdvanced/_1Lab/_04PersonalTitles.js
function personalTitles(age, gender) {
    let output = "";

    if (age >= 16) {
        if (gender === 'm') {
            output = "Mr.";
        } else if (gender === 'f') {
            output = "Ms.";
        }
    } else if (age < 16) {
        if (gender === 'm') {
            output = "Master";
        } else if (gender === 'f') {
            output = "Miss";
        }
    }

    console.log(`${output}`)
}